common/test/test_microsites/test_microsite/templates/static_templates/copyright.html

```diff
@@ -0,0 +1 @@
+This is copyright page of edX.
```

Comment:
Nit:  Reword to something along the lines of "This is a copyright page for an Open edX microsite"